src/Filter/ChoiceFilter.php

```diff
@@ -54,9 +54,9 @@ public function getRenderSettings(): array
     /**
      * @param mixed[] $data
      *
-     * @phpstan-param array{type?: string|int|null, value?: mixed} $data
+     * @phpstan-param array{type: string|int|null, value: mixed} $data
```

Comment:
Can `type` be a string? I'm assuming only `int` in https://github.com/sonata-project/SonataAdminBundle/pull/7022

Reply:
Yes I need to update it